packages/gatsby/cache-dir/static-entry.js

```diff
@@ -60,7 +60,7 @@ export default (pagePath, callback) => {
 
   let bodyHtml = ``
   let headComponents = []
-  let htmlAttributes = {}
+  let htmlAttributes = {xmlns: `http://www.w3.org/1999/xhtml`}
```

Comment:
I don't have strong feeling about this - but this could be added via plugin in `gatsby-ssr.js` in `onRenderBody` -> `setHtmlAttributes` - not sure what are pros and cons of defaulting to using that

Reply:
Thanks for that info, I hadn't thought of using `gatsby-ssr.js` in that way.